lib/api/filter.rb

```diff
@@ -43,13 +42,22 @@ def parse
         end
 
         associations.map! { |assoc| ".#{assoc}" }
-        field = "#{model.name}#{associations.join}-#{attr}"
-        target = parsed_filter[:logical_or] ? or_expressions : and_expressions
-        target << {parsed_filter[:operator] => {"field" => field, "value" => parsed_filter[:value]}}
+
+        field    = "#{model.name}#{associations.join}-#{attr}"
+        operator = parsed_filter[:operator]
```

Comment:
nit: can we just use `parsed_filter[:operator]` in 2 places rather than introducing a variable?
(that way `field` will be an unchanged line as well)

Reply:
Honestly, the lines were just getting very long with that didn't want to keep typing out `parsed_filter` when building these lines (where it made sense).

I could change it to `op` so that the `field = ...` line stays the same length though (effectively keeping it unchanged as well).

Reply:
changing to `op` may freak out rubocop. keep as is.

Reply:
@kbrock uhhh... already changed, and `rubocop` seems cool with it.

Reply:
thank you rubocop for being kind to my friend